src/pytest_regressions/data_regression.py

```diff
@@ -46,13 +48,17 @@ def check(
             will ignore ``datadir`` fixture when reading *expected* files but will still use it to
             write *obtained* files. Useful if a reference file is located in the session data dir for example.
 
+        :param prescision: if given, round all floats in the dict to the given number of digits.
+
         ``basename`` and ``fullpath`` are exclusive.
         """
         __tracebackhide__ = True
 
+        if prescision is not None:
+            data_dict = round_digits(data_dict, prescision)
```

Comment:
Let's not use the return value here, to make it more clear that we are mutating the value in-place.

```suggestion
            round_digits(data_dict, prescision)
```